src/extension.ts

```diff
@@ -285,7 +285,7 @@ enum DeviceState {
 }
 
 class DeviceTreeItem extends vscode.TreeItem {
-    rootDirectory : File;
+    private rootDirectory : File;

```

Comment:
FYI, this needs to be public. I didn't notice until after I merged and got a compile error.